Source/FAQView.swift

```diff
@@ -111,6 +110,24 @@ public class FAQView: UIView {
     }
   }
   
+  public var dataDetectorTypes: UIDataDetectorTypes? {
```

Comment:
Can you add any examples on README where this can be used? So that it will be easier for anybody to use this property.